data/basics/teal.go

```diff
@@ -239,6 +239,14 @@ func (tt TealType) String() string {
 	return "?"
 }
 
+// TealTypeFromString converts "b"/"u" string back to TealType
+func TealTypeFromString(t string) TealType {
+	if t == "b" {
+		return TealBytesType
+	}
+	return TealUintType
```

Comment:
Shouldn't this produce an error if the string is neither `"b"` or `"u"`?

Reply:
This is similar to argType in `logic` package - if not byte then it, so it might be ok.

Reply:
I was able to just delete this method entirely, it's unused.